webanno/webanno
# #1197 - codebook annotation support

**What's in the PR**
*  removed codebook dependencies from TagSetSelectionPanel by
  introducing CodebookCategory and CodebookTag as well as codebook
  internal CodebookTagEditorPanel and CodebookTagSelectionPanel

* Please note that this PR includes a lot of code duplication from core code since we thought of this as the only possible way to remove the dependency in TagSetSelectionPanel

**How to test manually**
* Not visible in the UI. You have to check the DB to see the changes (i.e. 2 new tables)

**Automatic testing**

**Documentation**
